.github/pull_request_template.md

```diff
@@ -8,22 +8,19 @@
 
 ## Related Issue
 
-```{=html}
-<!--- if this closes an issue make sure include e.g., "closes #4"
-or similar - or if just relates to an issue make sure to mention
-it like "#4" -->```
+`{=html} <!--- if this closes an issue make sure include e.g., "closes #4" or similar - or if just relates to an issue make sure to mention it like "#4" -->`
+
 ## Task list
 
-```{=html}
-<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists
-for an explanation on how to use task lists-->```
+`{=html} <!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists for an explanation on how to use task lists-->`
+
 Steps by contributor:
 
--   [ ] Add description to this pull request (under "\#\# Description")
+-   [ ] Add description to this pull request (under "## Description")
 -   [ ] Submit the pull request by clicking 'create pull request'
 -   [ ] Mark the pull request as draft
```

Comment:
To be honest, I never tried this before, so I checked it right now, and I see no option to convert a PR to draft before clicking 'create pull request'.  I suppose this option only appears after the PR is created?

Reply:
I've rechecked; it can be done in a dropdown:

![afbeelding](https://user-images.githubusercontent.com/19164640/183054117-d046129b-635e-4394-b526-4570e3b00f1a.png)

Of course both approaches are valid, I just like this one more :yum:

Reply:
I suppose many people don't know this indeed. Anyway your suggestion to collapse them in one item that (just shortly) refers the 2 possibilities, is very reasonable; feel free to add that :+1:.